shell/platform/windows/flutter_windows_view.cc

```diff
@@ -8,6 +8,16 @@
 
 namespace flutter {
 
+static bool IsValidResize(size_t cur_width,
```

Comment:
Needs a comment.

Also, I'm not sure "Valid" is really the right descriptor here. The OS is communicating something valid with a 0x0 resize.

Reply:
Changed this to `SurfaceWillUpdate`, I am not a huge fan of it since the name expects the embedding to know when the engine will update the surface to synchronize resize events, open to alternative suggestions.